manifests/d/Datto/BackupAgent/3.0.3.0/Datto.BackupAgent.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=MDSU.5-1-18362-1801
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Datto.BackupAgent
+PackageVersion: 3.0.3.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /passive
+  Custom: /norestart
+InstallerSuccessCodes:
+- 3010
+UpgradeBehavior: install
+ProductCode: '{3862CAAC-9FF6-409C-8F29-49FE503405C9}'
```

```suggestion
ProductCode: '{910c4dd5-1519-4423-8292-7f4cb168617b}'
```
`winget uninstall "{3862CAAC-9FF6-409C-8F29-49FE503405C9}"` does not work